### ISSUE TITLE
MergedGridSource mipmap shift

### ISSUE THREAD

**Issue by tischi** [mobie/mobie-viewer-fiji#687]
There still is a little jump when zooming in and out from the mipmaps, e.g. visible in 

```
new MoBIE( "https://github.com/mobie/covid-if-project", MoBIESettings.settings().gitProjectBranch( "main" ).imageDataFormat( ImageDataFormat.OmeZarrS3 ).view( "boundary-segment-test" ) ).close();
```

**Comment by tischi** [comment]
made branch `issue687` to work on this.

**Comment by tischi** [comment]
I think the general issue is the down-sampling padding.

Let's say we have images with 5 voxels; if we down-sample by two we have 3 voxels (if we do not want to loose information).  

Example: Stitching of three images:

level 0: 5    +  5   +  5  (= 15)
level 1:  2*3 + 2*3 + 2*3 (= 18)  (3 voxels and 2 times up-sampling)

The tricky thing is that while for the first image everything renders correctly, the more images one stitches the worse it gets. This is also what I see for the plate view, i.e. at the top left it is OK, but bottom right is bad.

Maybe we can fix this by means of the tile margin.
In the example above, we would need to add 1 voxel more to the tile margin of the level 0.
Then maybe it would fit again.

@constantinpape do you have an idea, can you even follow my logic?

**Comment by constantinpape** [comment]
I think what you write makes sense @tischi. Since we will have downsamplings by factors of two in almost all cases, can we fix it by choosing margins / paddings that are a factor of 2?

**Comment by tischi** [comment]
Should be fixed in `3.0.6`.
